buildmultiplescalaversions.sh

```diff
@@ -4,4 +4,7 @@ set -eu
 mvn "$@"
 ./change-scala-versions.sh 2.11
 mvn "$@"
+./change-spark-versions.sh 2
```

Comment:
So, same thing as what I said [there](https://github.com/deeplearning4j/DataVec/pull/171#discussion_r100231068) basically : the current state of the repo is a default of scala 2.11, and this "buildmultiple" script should reflect that.